src/netif.c

```diff
@@ -1192,6 +1196,7 @@ static void config_lcores(struct list_head *worker_list)
         else
             lcore_conf[id].type = LCORE_ROLE_IDLE;
 
+        lcore2index[worker_min->cpu_id] = id;
```

Comment:
`worker_min->cpu_id`  is from user's config file , and is possibly out of bound of array `lcore2index`.